velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -32,15 +32,21 @@ class SimpleAggregatesTest : public WindowTestBase {
             size, [](auto row) -> int32_t { return row % 10; }),
         makeFlatVector<int32_t>(
             size, [](auto row) -> int32_t { return row % 7; }),
-        makeFlatVector<int32_t>(size, [](auto row) -> int32_t { return row; }),
+        // k in k PRECEDING / FOLLOWING frame bounds, represented by c2 in
+        // certain testcases, should at least be 1.
+        makeFlatVector<int32_t>(
```

Comment:
This needs to be int64_t vector. int32_t could run into errors here since the code internally use only in64_t vectors for k.